package.json

```diff
@@ -197,12 +197,12 @@
     "mocha": "^10.4.0",
     "mocha-github-actions-reporter": "^0.3.1",
     "mocha-junit-reporter": "^2.2.1",
-    "node-sass": "^9.0.0",
     "null-loader": "^4.0.1",
     "prettier": "^3.2.5",
     "react-hot-loader": "^4.13.0",
     "remark": "^15.0.1",
     "resolve-url-loader": "^5.0.0",
+    "sass": "^1.77.4",
```

Comment:
`node-sass` is deprecated and was breaking CI because of this issue for one of its dependencies: https://github.com/nodejs/node-gyp/issues/2869
`sass` is the newer, active project.